handler/PostMessageSentHandler.ts

```diff
@@ -159,7 +159,7 @@ export class PostMessageSentHandler {
         const data = await retrieveDataByAssociation(read, getRoomAssoc(rid));
         if (DialogflowEnableChatClosedByVisitorEvent) {
             try {
-                const defaultLanguageCode = await getAppSettingValue(this.read, AppSetting.DialogflowDefaultLanguage);
+                const defaultLanguageCode = await Dialogflow.getLivechatAgentCredentials(read, rid, 'agent_default_language');
```

Comment:
We should create a separate enum for agent config this  i.e `agent_default_language`  Instead of using string

Reply:
@Shailesh351 How about we use enum LanguageCode in `enum/Dialogflow.ts`?

Result: `languageCode: data.custom_languageCode || LanguageCode.,defaultLanguageCode || LanguageCode.EN,`